pyulog/core.py

```diff
@@ -605,11 +605,10 @@ def _find_sync(self, last_n_bytes=-1):
                 current_file_position = self._file_handle.seek(current_file_position - len(chunk)\
                          + chunk_index + len(ULog.SYNC_BYTES), 0)
                 sync_seq_found = True
-                break
```

Comment:
This changes the behavior. The check advises to change `elif last_n_bytes != -1:` to `if last_n_bytes != -1:`

Reply:
Yes, sure; sorry... in next commit it is repaired.